### PR TITLE
Portfolio project

### PR DESCRIPTION
**Pull request by storres20**
# In this milestone I did the following activities:
- I create the repository "portfolio-project"
- I create the HTML and CSS files
- I add images for background, social-media and toggle menu bar (hamburger icon)
- I add Readme.md
- I add the linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <nav>
+    <h1 class="myLogo">My Logo</h1>
+    <img src="./img/Hamburger.png" alt="hamburger">
+  </nav>
+  
+  <header>
+    <h1 class="headerTitle">I'm Mariane Glad to see you!</h1>
+    <p class="headerParra">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+    <h2 class="headerSubtitle">LET'S CONNECT</h2>
+    <img src="./img/socialMobile.png" alt="socialMobile">
```
Comment (mgmediaweb) [inline index.html:20]: Kindly, I suggest you use the icons independently and list them with UL/LI

File: index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  <title>Document</title>
+</head>
+<body>
+  <nav>
+    <h1 class="myLogo">My Logo</h1>
+    <img src="./img/Hamburger.png" alt="hamburger">
+  </nav>
+  
+  <header>
+    <h1 class="headerTitle">I'm Mariane Glad to see you!</h1>
```
Comment (mgmediaweb) [inline index.html:17]: Kindly, I suggest you personalize this section with your information.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+  <title>Document</title>
+</head>
+<body>
+  <nav>
+    <h1 class="myLogo">My Logo</h1>
```
Comment (mgmediaweb) [inline index.html:12]: Kindly, I suggest you personalize this section with your information (your name).

File: index.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+  <title>Document</title>
```
Comment (mgmediaweb) [inline index.html:8]: Kindly, I suggest you put an appropriate title to your website

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  color: #6070ff;
+}
+
+header {
+  background: #fff;
+  border-bottom-left-radius: 80px;
+  background-image: url("./img/header-bg.png");
+  background-repeat: no-repeat;
+  background-size: 100%;
+  padding: 100px 24px 120px 24px;
```
Comment (mgmediaweb) [inline styles.css:33]: Kindly I suggest you to use flexbox to center elements in the container

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @storres20, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost5


## Highlights 💯 
- Professional-looking readme file. ✔️ 
- Clear and descriptive Pull Request. ✔️ 
- Clean and tidy code. ✔️ 
- Good job following the design. ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alfredbis29 — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @storres20 ,

✔️ Good documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**